actions/admin/models.go

```diff
@@ -38,11 +38,18 @@ type CreateXpub struct {
 	Key string `json:"key" example:"xpub661MyMwAqRbcGpZVrSHU..."`
 }
 
+// CreateUser is the model for creating a user
+type CreateUser struct {
+	// Accepts a JSON object for embedding custom metadata, enabling arbitrary additional information to be associated with the resource
+	Metadata engine.Metadata `json:"metadata" swaggertype:"object,string" example:"key:value,key2:value2"`
+	// The xpub key
+	Key string `json:"key" example:"xpub661MyMwAqRbcGpZVrSHU..."`
```

Comment:
Maybe we could rename it to be something like XPubKey or something indicating that it is actually a public key and not a private one? WDYT?

Reply:
ok you see i forgot to remove it. `CreateUser`